src/main/java/Duke.java

```diff
@@ -6,5 +7,80 @@ public static void main(String[] args) {
+            }
+
+            if (line.contains(" ")){
+                //int valIndex = Integer.parseInt(line.substring(line.indexOf(" ") + 1));
+                if ((line.substring(0, line.indexOf(" "))).equals("mark")) {
+                    int valIndex = Integer.parseInt(line.substring(line.indexOf(" ") + 1));
+                    if (valIndex <= inputCount){
+                        userInput[valIndex - 1].markAsDone();
+                        System.out.println("Nice! I've marked this task as done: ");
+                        System.out.println("[" + userInput[valIndex - 1].getStatusIcon() + "] " + userInput[valIndex - 1].description);
+                    }
+                    else {
+                        System.out.println("Invalid Command!");
+                    }
+                    continue;
+                }
+                else if ((line.substring(0, line.indexOf(" "))).equals("unmark")) {
+                    int valIndex = Integer.parseInt(line.substring(line.indexOf(" ") + 1));
+                    if (valIndex <= inputCount) {
+                        userInput[valIndex - 1].markAsNotDone();
+                        System.out.println("OK, I've marked this task as not done yet:");
+                        System.out.println("[" + userInput[valIndex - 1].getStatusIcon() + "] " + userInput[valIndex - 1].description);
+                    }
+                    else {
+                        System.out.println("Invalid Command!");
+                    }
+                    continue;
+                }
+                else if ((line.substring(0, line.indexOf(" "))).equals("Todo")){
+                    userInput[inputCount] = new Todo((line.substring(1 + line.indexOf(" "))));
+                    inputCount++;
+                    System.out.println("Got it. I've added this task: ");
+                    System.out.println("[" + userInput[inputCount - 1].getIcon() + "]" + "[" + userInput[inputCount - 1].getStatusIcon() + "] " + userInput[inputCount - 1].description);
+                    System.out.println("Now you have " + inputCount + " tasks in the list.");
+                    continue;
+                }
+                else if ((line.substring(0, line.indexOf(" "))).equals("Deadline")){
+                    userInput[inputCount] = new Deadline((line.substring(1 + line.indexOf(" "), line.indexOf("/by"))),line.substring(line.indexOf("/by") + 4));
+                    inputCount++;
+                    System.out.println("Got it. I've added this task: ");
+                    System.out.println("[" + userInput[inputCount - 1].getIcon() + "]" + "[" + userInput[inputCount - 1].getStatusIcon() + "] " + userInput[inputCount - 1].description + "(by: " + userInput[inputCount - 1].getBy() + ")");
+                    System.out.println("Now you have " + inputCount + " tasks in the list.");
+                    continue;
+                }
+                else if ((line.substring(0, line.indexOf(" "))).equals("Event")){
```

Comment:
Consider putting else if in the same line as the two braces as per k&r coding standard